website/project/__init__.py

```diff
@@ -42,7 +42,7 @@ def new_node(category, title, user, description=None, project=None):
     :return Node: Created node
 
     """
-    category = category.strip().lower()
+    category = '_'.join(category.strip().lower().split(' '))
```

Comment:
Definitely agree here. I want to change the category representation so we can query for it in elasticsearch without the label being tokenized into 'methods', 'and', 'measures'.

Probably better here is to update the UI select to have values that correspond with the backend representation. 

was trying to avoid the